setup.cfg

```diff
@@ -114,7 +114,7 @@ transformers =
 vllm =
     vllm>=0.2.6
 embedding =
-    sentence-transformers<2.3.0
```

Comment:
The `sentence-transformers` dependency is set in 3 places:
- python.yaml for CI
- all and embedding in setup.cfg